src/main/java/mate.academy/lib/Inject.java

```diff
@@ -1,5 +1,11 @@
 package mate.academy.lib;
 
-public @interface Inject {
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
 
+@Retention(RetentionPolicy.RUNTIME)
+@Target(value = {ElementType.FIELD})
```

Comment:
If you use only one value that enough: 
```suggestion
@Target(value = ElementType.FIELD)
```